recover-id-object/scripts/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG build_image=rust:1.70.0
+ARG base_image=debian:buster-slim
+FROM ${build_image} AS build
+
+WORKDIR /usr/app/recover-id-object
+
+COPY recover-id-object/Cargo.toml recover-id-object/Cargo.lock .
+COPY deps /usr/app/deps
+RUN mkdir src && echo 'fn main() { println!("Dummy!"); }' > ./src/main.rs
+
+RUN cargo build --release --locked
```

Comment:
These should be ensured at the CI level that they are updated or they should be removed from the repo. 

So far lets use them and I will make the above in another PR